include/picolibrary/microchip/megaavr0/asynchronous_serial.h

```diff
@@ -68,6 +71,225 @@ enum class USART_Clock_Generator_Operating_Speed : std::uint8_t {
+     */
+    void initialize() noexcept
+    {
+        configure_txd_pin_as_output();
+
+        enable_transmitter();
+    }
+
+    /**
+     * \brief Transmit data.
+     *
+     * \param[in] data The data to transmit.
+     */
+    void transmit( Data data ) noexcept
+    {
+        while ( not transmit_buffer_is_empty() ) {} // while
+
+        load_transmit_buffer( data );
+    }
+
+  private:
+    /**
+     * \brief The USART used by the transmitter.
+     */
+    Peripheral::USART * m_usart{};
+
+    /**
+     * \brief Disable the transmitter.
+     */
+    constexpr void disable() noexcept
+    {
+        if ( m_usart ) {
+            disable_transmitter();
+        } // if
+    }
+
+    /**
+     * Configure the transmitter.
+     *
+     * \param[in] usart_data_bits The desired USART data bits configuration.
+     * \param[in] usart_parity The desired USART parity configuration.
+     * \param[in] usart_stop_bits The desired USART stop bits configuration.
+     * \param[in] usart_clock_generator_operating_speed The desired USART clock generator
+     *            operating speed configuration.
+     * \param[in] usart_clock_generator_scaling_factor The desired USART clock generator
+     *            scaling factor (BAUD register value).
+     */
+    void configure_transmitter(
+        USART_Data_Bits                       usart_data_bits,
+        USART_Parity                          usart_parity,
+        USART_Stop_Bits                       usart_stop_bits,
+        USART_Clock_Generator_Operating_Speed usart_clock_generator_operating_speed,
+        std::uint16_t usart_clock_generator_scaling_factor ) noexcept
+    {
+        m_usart->ctrlb = static_cast<std::uint8_t>( usart_clock_generator_operating_speed );
+        m_usart->ctrla = 0;
+        m_usart->ctrlc = Peripheral::USART::CTRLC::CMODE_ASYNCHRONOUS
+                         | static_cast<std::uint8_t>( usart_data_bits )
+                         | static_cast<std::uint8_t>( usart_parity )
+                         | static_cast<std::uint8_t>( usart_stop_bits );
+        m_usart->status = Peripheral::USART::STATUS::Mask::TXCIF
+                          | Peripheral::USART::STATUS::Mask::RXSIF
+                          | Peripheral::USART::STATUS::Mask::ISFIF
+                          | Peripheral::USART::STATUS::Mask::BDF;
+        m_usart->baud = usart_clock_generator_scaling_factor;
+    }
+
+    /**
+     * \brief Disable the transmitter.
+     */
+    void disable_transmitter() noexcept
+    {
+        m_usart->ctrlb &= ~Peripheral::USART::CTRLB::Mask::TXEN;
+
+        // Silicon errata workaround ("TXD Pin Override Not Released When Disabling the
+        // Transmitter")
+        m_usart->status = 0;
+    }
+
+    /**
+     * \brief Configure the TXD pin to act as an output.
+     */
+    void configure_txd_pin_as_output() noexcept
+    {
+        Multiplexed_Signals::txd_port( *m_usart ).dirset = Multiplexed_Signals::txd_mask( *m_usart );
```

Comment:
The TXD pin should be configured to be high when it is configured to be an output.